words/predicates.txt

```diff
@@ -278,44 +325,54 @@ deserted
 destiny
 detailed
 determined
+developing
+diagnostic
 diamond
 different
+difficult
 diligent
 dirt
 disco
+discovered
 discreet
 distinct
 dog
 dolomite
+dorian
 dot
 dour
 dramatic
 dull
 dune
+dusky
```

Comment:
I've seen this used as an alternative to "dark-skinned". Probably outdated, but maybe remove?